src/solver/optimisation/base_weekly_optimization.cpp

```diff
@@ -19,10 +19,12 @@
 ** along with Antares_Simulator. If not, see <https://opensource.org/license/mpl-2-0/>.
 */
 
-#include <memory>
 #include "antares/solver/optimisation/base_weekly_optimization.h"
-#include "antares/solver/optimisation/weekly_optimization.h"
+
+#include <memory>
```

Comment:
External inclusion **\<memory\>** could come become internal inclusion

Reply:
Not sure I understand, this is a standard library header si I think it's expected to have it here.